### PR TITLE
zh: add the section "Custom Operator"

### PR DESCRIPTION
**Pull request by KanShiori**
### What is changed, added, or deleted? (Required)

Add the section "Custom Operator" in [deploy-tidb-operator.md](https://github.com/pingcap/docs-tidb-operator/blob/master/zh/deploy-tidb-operator.md).

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by csuzhangxc — APPROVED** [review]
LGTM

**Comment by KanShiori** [comment]
@qiancai  PTAL

File: zh/deploy-tidb-operator.md [hunk 1]
```diff
@@ -251,3 +251,22 @@ tidbmonitors.pingcap.com             2020-06-11T07:59:41Z
...
+## 配置 TiDB Operator
+
+可以通过修改 `/home/tidb/tidb-operator/values-tidb-operator.yaml` 来配置 TiDB Operator。本节后续使用 `values.yaml` 来代表 `/home/tidb/tidb-operator/values-tidb-operator.yaml`。
+
+TiDB Operator 包含两个组件：
+
+* tidb-controller-manager
+* tidb-scheduler
+
+这两个组件都是无状态的，由 `Deployment` 部署。在 `values.yaml` 文件中，你可以配置其中的 `limit`、`request` 与 `replicas` 参数。
```
Comment (qiancai) [inline zh/deploy-tidb-operator.md:264]: @csuzhangxc 可以请帮忙看看这句话里 “由 `Deployment` 部署” 中的 `Deployment` 具体指的是什么吗？

这里和英文是一致的。
These two components are stateless and deployed via `Deployment`.

File: zh/deploy-tidb-operator.md [hunk 2]
```diff
@@ -251,3 +251,22 @@ tidbmonitors.pingcap.com             2020-06-11T07:59:41Z
...
+
+修改了 `values.yaml` 文件后，运行下面命令来应用：
```
Comment (qiancai) [inline zh/deploy-tidb-operator.md:266]:
```suggestion
修改了 `values.yaml` 文件后，请运行以下命令使更改生效：
```

File: zh/deploy-tidb-operator.md (hunk 1 above)
Comment (csuzhangxc) [reply zh/deploy-tidb-operator.md:264]: K8s 里一种资源，所以我觉得 ``` `Deployment` ``` 这样标准应该够了？

File: zh/deploy-tidb-operator.md [hunk 3]
```diff
@@ -251,3 +251,22 @@ tidbmonitors.pingcap.com             2020-06-11T07:59:41Z
     ```shell
     helm upgrade tidb-operator ./tidb-operator
     ```
+
+## 自定义配置 TiDB Operator
+
+可以通过修改 `/home/tidb/tidb-operator/values-tidb-operator.yaml` 来配置 TiDB Operator。本节后续使用 `values.yaml` 来代表 `/home/tidb/tidb-operator/values-tidb-operator.yaml`。
```
Comment (DanielZhangQD) [inline zh/deploy-tidb-operator.md:257]: `/home/tidb` -> `${HOME}`, 英文那里也改下

Comment (KanShiori) [reply zh/deploy-tidb-operator.md]: Done

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

**Pull request merged**